stubs/_blueman.pyi

```diff
@@ -0,0 +1,58 @@
+from typing import Any, List, Dict, Union, Callable
+from mypy_extensions import TypedDict
+
+ERR: Dict[int, str]
+RFCOMM_HANGUP_NOW: int
+RFCOMM_RELEASE_ONHUP: int
+RFCOMM_REUSE_DLC: int
+RFCOMM_STATES: List[str]
+RFCOMM_TTY_ATTACHED: int
+
+class RfcommDev(TypedDict):
+    id: int
+    channel: int
+    flags: int
+    state: str
+    src: str
+    dst: str
+
+class HciInfo(TypedDict):
+    acl_mtu: int
+    acl_plts: int
+    bdaddr: str
+    dev_id: int
+    features: List[int]
+    flags: int
+    link_mode: int
+    link_policy: int
+    name: str
+    pkt_type: int
+    sco_mtu: int
+    stat: Dict[str, int]
+    type: int
+
+class BridgeException(Exception):
+    errno: int
+    def __init__(self, errno: int) -> None: ...
+
+class ConnInfoReadError(Exception): ...
+
+class RFCOMMError(Exception): ...
+
+class conn_info:
+    failed: bool
+    def __init__(self, addr: str, hci_name: str) -> None: ...
+    def deinit(self) -> None: ...
+    def get_lq(self) -> int: ...
+    def get_rssi(self) -> int: ...
+    def get_tpl(self) -> int: ...
+    def init(self) -> None: ...
+
+def create_bridge(name: str) -> None: ...
```

Comment:
optional

Reply:
See comment earlier, Union[str, str] does not make sense to me.